src/Sylius/Bundle/TaxonomyBundle/Doctrine/ORM/TaxonRepository.php

```diff
@@ -110,4 +110,24 @@ public function createListQueryBuilder()
     {
         return $this->createQueryBuilder('o')->leftJoin('o.translations', 'translation');
     }
+
+    /**
+     * @param QueryBuilder $qb
+     * @param null $locale
+     * @return QueryBuilder
+     */
+    private function createTranslationQueryBuilderPart(QueryBuilder $qb, $locale = null)
+    {
+        if ($locale !== null) {
+            $qb
```

Comment:
I know, that `qb` is pretty obvious, but we are not allowing abbreviation as variable names. Please, change to `queryBuilder` here and above.